src/NUnitFramework/framework/Internal/TestProgressReporter.cs

```diff
@@ -11,19 +14,22 @@ namespace NUnit.Framework.Internal
     /// the async callbacks that are used to inform the client
     /// software about the progress of a test run.
     /// </summary>
-    public class TestProgressReporter : ITestListener
+    public sealed class TestProgressReporter : ITestListener
     {
-        static readonly Logger log = InternalTrace.GetLogger("TestProgressReporter");
+        private static readonly Logger log = InternalTrace.GetLogger("TestProgressReporter");
```

Comment:
Thanks for making the visibility explicit

Reply:
We should enable the StyleCop rule and do it for all fields.

Reply:
I'm also partly confused about usage of regions (I personally don't like them) and braces (I usually always use them), maybe they could be part of the rule too if either one should not be used.